cmd/state-svc/internal/rtwatcher/watcher.go

```diff
@@ -121,4 +121,5 @@ func (w *Watcher) Watch(pid int, exec string, dims *dimensions.Values) {
 	dims.Sequence = p.IntP(-1) // sequence is meaningless for heartbeat events
 	e := entry{pid, exec, dims}
 	w.watching = append(w.watching, e)
+	go w.RecordUsage(e) // initial event
```

Comment:
Instead of emitting the analytics event in the location mentioned by the ticket: https://sourcegraph.com/github.com/ActiveState/cli@d8d2fa6c465c7b8e8456e02ca1bcdd2b1cc77423/-/blob/cmd/state-svc/internal/resolver/resolver.go?L197:20&popover=pinned, emit it from the function called by that function: https://sourcegraph.com/github.com/ActiveState/cli@d8d2fa6c465c7b8e8456e02ca1bcdd2b1cc77423/-/blob/cmd/state-svc/internal/resolver/resolver.go?L206